JMS/com/ibm/mq/samples/jms/JmsResponse.java

```diff
@@ -65,7 +65,7 @@ public static void main(String[] args) {
         setJMSProperties(connectionFactory);

```

Comment:
Copyright date for each modified file needs to be updated. eg. 
from 
````
(c) Copyright IBM Corporation 2019
````
to
````
(c) Copyright IBM Corporation 2019, 2022
````